getting-started/exploring-forge.md

```diff
@@ -29,10 +29,10 @@ and, failing that, please [contact us][contact-timescale].
 <img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/forge_images/timescale-forge-service-creation.png" alt="Set up a Timescale Forge service"/>
 
 1. First, supply your service name (e.g., `acmecorp-test` or `acmecorp-dev`).
-1. Next, choose your CPU and memory configuration, from (0.5 CPU, 2GB RAM) to 
```

Comment:
sorry, wasn't clear about that. Let's stick with the 0.5 compute recommendation but change Storage to 10GB. Like in the UI.